net-dns/yaddns/yaddns-1.1.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="A small dynamic dns client with multiple servers support and a high flexibility"
+DESCRIPTION="Yaddns is a small dynamic dns client with multiple servers support and a high flexibility."
+HOMEPAGE="https://yaddns.github.io/"
+COPYRIGHT="2015 Anthony Viallard and Raphael Huck"
```

Comment:
We would need some discussion on this. It seems to me that there're only [some files][0] with copyright attributed to "Anthony". Should we only assign him to copyright as he seem to be the only person who claim it? Also, the copyright years seems to be 2008 and 2009.

Also, UTF-8 is fine here, so you could use "Raphaël Huck" instead of "Raphael Huck"

```diff
-COPYRIGHT="2015 Anthony Viallard and Raphael Huck"
+COPYRIGHT="2008, 2009 Anthony Viallard"
```

[0]: https://github.com/yaddns/yaddns/search?p=1&q=copyright&type=&utf8=%E2%9C%93

Reply:
Of course, if you would want to credit the authors, installing the `AUTHORS` file would be suffice